src/index.html

```diff
@@ -22,6 +22,335 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
```

Comment:
You need to add class with BEM
```suggestion
    <header class="header">
      <a
        href="#"
        class="header__link"
      >
        <img
          class="header__logo"
          src="images/logo.png"
          alt="Moyo logo"
        />
```